script/installation/packages.sh

```diff
@@ -120,19 +120,16 @@ if [ "$DISTRO" = "UBUNTU" ]; then
     if [ "$MAJOR_VER" == "14" ]; then
         PKG_CMAKE="cmake3"
         FORCE_Y="--force-yes"
-	TF_VERSION="1.4.0"
-        TFBinaryURL="https://storage.googleapis.com/tensorflow/linux/${TF_TYPE}/tensorflow-${TF_VERSION}-cp34-cp34m-linux_x86_64.whl"
+	    TF_VERSION="1.4.0"
```

Comment:
Very minor change - just fixing the indentation on all the `TF_VERSION` lines.

Reply:
Oh, I should have changed indentation setting in my editor earlier.